source/options.html

```diff
@@ -23,6 +23,13 @@ <h2 class="only-firefox">Options</h2>
 		</label>
 	</p>
 
+	<p>
+		<label>
+			<strong>Personal Token</strong> (for high rate API calls and private repos)<br>
```

Comment:
This should link directly to getting a token: https://github.com/settings/tokens/new?description=Refined%20GitHub